src/Illuminate/Support/Arr.php

```diff
@@ -525,4 +525,28 @@ public static function where($array, callable $callback)
 
         return $filtered;
     }
+
+    /**
+     * Get the array key for the given value.
+     *
+     * @param array $array
+     * @param mixed $value
+     * @return mixed
+     */
+    public static function key($array, $value)
+    {
+        $lookFor = $value;
```

Comment:
@lucasmichot is right. using `array_search` makes more sense. I'd suggest to return `false` explicitely rather than `null`.